xbmc/epg/EpgContainer.h

```diff
@@ -281,7 +283,7 @@ namespace EPG
 
     void InsertFromDatabase(int iEpgID, const std::string &strName, const std::string &strScraperName);
 
-    typedef std::map<unsigned int, CEpg*> EPGMAP;
+    typedef std::map<unsigned int, std::shared_ptr<CEpg>> EPGMAP;
     typedef EPGMAP::iterator              EPGMAP_ITR;
```

Comment:
In a slightly unrelated note these could be removed in favor of using `auto` at some point.